cves/CVE-2004-0811.yml

```diff
@@ -95,9 +100,9 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
+  answer: Discovered by a user running a simple test script to exercise a theory they had regarding request authentication.
+  date: 2002-11-21
+  automated: false
   google:
```

Comment:
Maybe add false for the "google" and "contest" fields just for more clarity for a reader

Reply:
Sure, sounds reasonable.